event_webhook_types.go

```diff
@@ -609,6 +616,18 @@ type MergeEvent struct {
 			Previous []*EventLabel `json:"previous"`
 			Current  []*EventLabel `json:"current"`
 		} `json:"labels"`
+		LastEditedAt struct {
+			Previous string `json:"previous"`
+			Current  string `json:"current"`
+		} `json:"last_edited_at"`
+		LastEditedByID struct {
+			Previous int `json:"previous"`
+			Current  int `json:"current"`
+		} `json:"last_edited_by_id"`
+		MilestoneID struct {
+			Previous int `json:"previous"`
+			Current  int `json:"current"`
+		} `json:"milestone_id"`
```

Comment:
I don't see these fields in the docs for merge events, only for issue events...

Reply:
Cool, let me know when it's merged so I can have another look 👍🏻

Reply:
@svanharmelen It is merged.